### ISSUE TITLE
cannot save case in DesighSphysics

### ISSUE THREAD

**Issue by WangJJ666** [DualSPHysics/DesignSPHysics#30]
When I use the  version of DesighSphysics-develop, I get an error that the case cannot be saved, it says: 'gbk' codec can't decode byte 0x93 in position 2672: illegal multibyte sequence

**Comment by zlotny** [comment]
Again, I'm assuming you closed the issue because you found a solution for your problem. 

Basically you wrote some text that needed the GBK character encoding (basically, simplified Chinese characters). However, there's not much I can do about that without the full stack trace :(

That exception could have arisen in a lot of places in the code. 

If you keep having this problem please include all the error text on the error window.

**Comment by WangJJ666** [comment]
Hello, thank you very much for your reply. I downloaded the file you just updated and Acceleration Input is ready to use, and the case can be saved. But when I clicked run GenCase, I got an error again, here is the detailed description:
DesignSPHysics version: 0.6.0.1912-03-4
Platform: Windows-10-10.0.17763
FreeCAD Version: 0.18.4 (GitTag)
Exception type: <class 'IndexError'&gt;
Exception value: list index out of range
Traceback:
&nbsp; File "C:\Users\dell\AppData\Roaming\FreeCAD\Mod\DesignSPHysics-master\mod\widgets\dock\dock_pre_processing_widget.py", line 211, in on_execute_gencase
&nbsp; &nbsp; output = str(process.readAllStandardOutput().data(), encoding='utf-8').split("================================\n")[1]





------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Andrés Vieira"<notifications@github.com&gt;;
发送时间:&nbsp;2019年12月3日(星期二) 下午4:50
收件人:&nbsp;"DualSPHysics/DesignSPHysics"<DesignSPHysics@noreply.github.com&gt;;
抄送:&nbsp;"王佳静"<1175981439@qq.com&gt;;"State change"<state_change@noreply.github.com&gt;;
主题:&nbsp;Re: [DualSPHysics/DesignSPHysics] cannot save case in DesighSphysics (#30)




Again, I'm assuming you closed the issue because you found a solution for your problem.
 
Basically you wrote some text that needed the GBK character encoding (basically, simplified Chinese characters). However, there's not much I can do about that without the full stack trace :(
 
That exception could have arisen in a lot of places in the code.
 
If you keep having this problem please include all the error text on the error window.

**Comment by zlotny** [comment]
Hi there. We found that error today on Windows systems. It is already fixed on the master branch. Please update your installation and you should be good to go :+1:

**Comment by WangJJ666** [comment]
Hi, I'm really sorry to bother you again. When I created geometries in FreeCAD according to the CasePistonDamping_IRREG_Def.xml in the examples folder of DualSPHysics_v4.4, I encountered this problem:

*** Exception (JXml::ErrReadElement)

Text: Error reading xml - Element 'spectrum' is invalid.

File: E:/DualSPHysicsProjects/1206-damping-irreg-test/1206-damping-irreg-test_out/1206-damping-irreg-test.xml(row:82)

Finished execution (code=1).




I don't know why this happens because my settings are the same as in the example.&nbsp;I really need your help, and thank you very much.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"Andrés Vieira"<notifications@github.com&gt;;
发送时间:&nbsp;2019年12月3日(星期二) 晚上7:23
收件人:&nbsp;"DualSPHysics/DesignSPHysics"<DesignSPHysics@noreply.github.com&gt;;
抄送:&nbsp;"王佳静"<1175981439@qq.com&gt;;"State change"<state_change@noreply.github.com&gt;;
主题:&nbsp;Re: [DualSPHysics/DesignSPHysics] cannot save case in DesighSphysics (#30)




Hi there. We found that error today on Windows systems. It is already fixed on the master branch. Please update your installation and you should be good to go

**Comment by zlotny** [comment]
Hi. It should be fixed now. Please update your DesignSPHysics version (Addon manager, download DesignSPHysics again).

However you will **need** to **delete the previous piston wave generator**.

![image](https://user-images.githubusercontent.com/6155245/70513965-eb680900-1b32-11ea-8eb7-b72e67d0ac41.png)

And then create a new one again. If you don't do this the problem will persist as the bug is saved on your case files on disk.


Also, please if you have further problems open a new issue. I didn't see this until now because **the issue is marked as closed**. 
If you mark issues as closed they do not appear in here:
![image](https://user-images.githubusercontent.com/6155245/70514045-1bafa780-1b33-11ea-809e-98a5b67d37a8.png)

So I can't know that the software has a bug. So please, do not further comment in here, instead open a new issue for a different problem.
